src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,20 @@
 package core.basesyntax;
 
+import enums.Colors;
+import java.util.Random;
+
 public class ColorSupplier {
+    private int colorIndex;
+
+    public ColorSupplier() {
+        this.colorIndex = new Random().nextInt(Colors.values().length);
+    }
+
     public String getRandomColor() {
-        return null;
+        return Colors.values()[getColorIndex()].toString();
+    }
+
+    public int getColorIndex() {
+        return colorIndex;
     }
```

Comment:
we dont need that method. instead, calculate `colorIndex` right in the `getRandomColor` method
```suggestion
```

Reply:
++